### PR TITLE
Matrixfilter NA options

### PR DESCRIPTION
**Pull request by WackerO**
When updating the matrixfilter module with the NA options, I totally forgot to also add them to the pipeline to make them actually callable; sneaking in this small PR to do that now.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -463,8 +463,10 @@ process {
             "--sample_id_col \"${params.observations_id_col}\"",
             "--minimum_samples ${params.filtering_min_samples}",
             "--minimum_abundance ${params.filtering_min_abundance}",
+            "--minimum_proportion_not_na ${params.filtering_min_proportion_not_na}",
```
Comment (pinin4fjords) [inline conf/modules.config:466]: This needs conditional treatment as well, right?

**Review by pinin4fjords — COMMENTED** [review]
Minor q on the option handling

File: conf/modules.config (hunk 1 above)
Comment (WackerO) [reply conf/modules.config:466]: No, there is a default value for that param in the nextflow.config. If it is provided together with the min sample number, the latter will be used in the module

Comment (pinin4fjords) [reply conf/modules.config:466]: What if someone sets the value to null?

Comment (WackerO) [reply conf/modules.config]: Ah, fair enough!

**Review by pinin4fjords — APPROVED** [review]

**Pull request merged**